File: URI/Lua/DividindoXPorY.lua
n = io.read("*number")
for i=1,n do
    x = io.read("*number")
    y = io.read("*number")
    if y == 0 then
        print("divisao impossivel")
    else
        print(string.format( "%.1f", x/y))
    end
end

File: URI/Python/DividindoXPorY.py
n = int(input())
for i in range(n):
    inf = [int(x) for x in input().split()]
    x = inf[0]
    y = inf[1]
    if y == 0:
        print("divisao impossivel")
    else:
        print("{:.1f}".format(x/y))